app/controllers/actions/AuthenticateAction.scala

```diff
@@ -53,7 +48,7 @@ private[actions] class AuthenticateAction @Inject()(override val authConnector:
         logger.warn(s"Missing channel header for request id ${request.headers.get("http_x_request_id")}")
         Future.successful(Left(BadRequest("Missing channel header or incorrect value specified in channel header")))
       case Some(channel) =>
-        implicit val hc: HeaderCarrier = HeaderCarrierConverter.fromHeadersAndSession(request.headers)
+        implicit val hc: HeaderCarrier = HeaderCarrierConverter.fromRequest(request)
```

Comment:
I think this might need to be fromRequestAndSession(request, request.session). At least that is what I have done. I don't know if it pulls that session out anyway? Might be worth checking, unless you already have.

Reply:
The default parameter for `session` in `fromHeadersAndSession` is `None`
<img width="764" alt="Screenshot 2021-09-15 at 10 01 18" src="https://user-images.githubusercontent.com/43777106/133404424-0e2bf1fe-ef5c-4848-8994-35655a254a2a.png">